spec/6819_show_box_info_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rspec'
+require 'spec_helper'
+
+describe 'test_spec' do
+  executeShellCommandsAndTestExitCode ([
```

Comment:
Please add testcase for 
` ./mdbci show box --box-name ubuntu_trusty_vbox `
and 
`./mdbci show box --box-name WRONG`